elasticsearch-transport/lib/elasticsearch/transport/client.rb

```diff
@@ -125,6 +125,11 @@ def perform_request(method, path, params={}, body=nil)
         transport.perform_request method, path, params, body
       end
 
+      # Shuts down the client
+      def close
+        @transport.__close_connections
```

Comment:
I agree __ should be for private methods, but here we're calling the private method from the outside of the `@transport`, right?